.github/workflows/ci.yaml

```diff
@@ -0,0 +1,37 @@
+name: Server CI
+
+on:  # yamllint disable-line rule:truthy
+  pull_request: {}
```

Comment:
```suggestion
  pull_request_target: {}
```

Reply:
Why do we need to target `pull_request_target`?  I don't think we would stand to gain from the extra privileges that `pull_request_target` would give us in this action.